psh/test-autocompletion.py

```diff
@@ -0,0 +1,114 @@
+# Phoenix-RTOS
+#
+# phoenix-rtos-tests
+#
+# psh sutocompletion and hints test
+#
+# Copyright 2021 Phoenix Systems
+# Author: Mateusz Niewiadomski
+#
+# This file is part of Phoenix-RTOS.
+#
+# %LICENSE%
+#
+import re
+
+from psh.tools.basic import run_psh, assert_only_prompt, assert_prompt
+
+
+TAB = '\t'
+ENTER = '\n'
+
+
+def mkdir(p, path):
+    p.send('mkdir ' + path + ENTER)
+    p.expect_exact('mkdir ' + path)
+    assert_prompt(p, msg='Cannot make directory: ' + path)
+
+
+def touch(p, path):
+    p.send('touch ' + path + ENTER)
+    p.expect_exact('touch ' + path)
+    assert_prompt(p, msg='Cannot make file: ' + path)
+
+
+def assert_completion(p, incomplete, complete):
+    p.send(incomplete)
+    p.expect_exact(incomplete)
+    p.send(TAB)
+    if complete.startswith(incomplete):
+        addition = re.sub('^' + incomplete, '', complete)
```

Comment:
```suggestion
        suffix = word[len(prefix):]
```
Thanks to that we do not need to import `re` submodule.

Reply:
Changed. Thanks!